### ISSUE TITLE
Error message on ECONNREFUSED is wrong when split queries are enabled

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#376]
Originally reported of Google Code with ID 320.

```
The error message mentions masterd, when actually it's hconfd.
```
Originally added on 2012-11-21 14:09:01 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-05-09 05:57:02 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
I actually fixed this a while back (commits 7f5edc60 and improved in 055d6ac0). The first was included in 2.7.0 beta2, don't know about the second one, but at least this can be marked fixed if not released.
```
Originally added on 2013-05-13 09:02:56 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-07-09 11:00:39 +0000 UTC.

Changed State: Released
Added to Milestone: Release2.7
